webrtc/src/api/media_engine/mod.rs

```diff
@@ -58,8 +56,21 @@ pub(crate) struct MediaEngineHeaderExtension {
     pub(crate) uri: String,
     pub(crate) is_audio: bool,
     pub(crate) is_video: bool,
-    // If set only Transceivers of this direction are allowed
-    pub(crate) allowed_directions: Vec<RTCRtpTransceiverDirection>,
+    pub(crate) allowed_direction: Option<RTCRtpTransceiverDirection>,
+}
+
+impl MediaEngineHeaderExtension {
+    pub fn is_matching_direction(&self, dir: RTCRtpTransceiverDirection) -> bool {
+        if let Some(allowed_direction) = self.allowed_direction {
+            use RTCRtpTransceiverDirection::*;
+            allowed_direction == Inactive && dir == Inactive
+                || allowed_direction.has_send() && dir.has_send()
```

Comment:
So this means `allowed_direction: SendRecv` matches all of `SendRecv`, `SendOnly`, and `RecvOnly`. That might be a bit unexpected and should probably be documented?

Reply:
That's how it worked-ish before though. Leaving the previous vector empty would have resulted in SendOnly+RecvOnly and a receiver/sender would have created both a SendOnly RecvOnly for the matching.

Reply:
Aight! Should we have a mini-migration guide in `webrtc/CHAGNELOG.md` to explain how to change?

Reply:
Ok!

Reply:
Done